.github/workflows/test-environment.yml

```diff
@@ -0,0 +1,127 @@
+name: sanity workflow
+
+on:
+  # Ability to execute on demand
+  workflow_dispatch:
+    inputs:
+      deployment_name:
+        type: string
+        description: |
+          Name your environment (Only a-zA-Z0-9 and `-`).
+          For example: john-8-7-2-June01'
+        required: true
+      elk-stack-version:
+        required: true
+        description: "Elastic Cloud stack version"
+        default: "8.7.2-SNAPSHOT"
+      ess-region:
+        required: true
+        description: "Elastic Cloud deployment region"
+        default: "gcp-us-west2"
+      cleanup-env:
+        description: "Boolean: Cleanup resources after provision"
+        default: false
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_REGION: "eu-west-3"
```

Comment:
Why would it be a good idea to use another region? I think it would make it easy for us to miss unutilized resources when we perform clean-ups. IMO we should try and stick to one region as where ever it is possible

Reply:
I just want to reduce false positive behavior. Human errors will always occur. Since we are allowing the environment to remain uncleared in this job, we can replicate the current state as it exists today in eu-west-1, where we have multiple deployed EKS clusters and EC2 machines. The main purpose of this job is to ensure stability over time, and one crucial aspect is to use a separate region.